forge-gui/res/cardsfolder/upcoming/kaylas_command.txt

```diff
@@ -0,0 +1,13 @@
+Name:Kayla's Command
+ManaCost:1 W W
+Types:Sorcery
+A:SP$ Charm | Choices$ DBToken,DBChoose,DBChangeZone,DBGainLife | CharmNum$ 2
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ c_2_2_a_construct | TokenOwner$ You | SpellDescription$ Create a 2/2 colorless Construct artifact creature token.
+SVar:DBChoose:DB$ ChooseCard | SubAbility$ DBCounter | Mandatory$ True | Choices$ Creature.YouCtrl | SpellDescription$ Put a +1/+1 counter on a creature you control. It gains double strike until end of turn.
```

Comment:
Why the ChooseCard line? Should be able to do this from PutCounter + `RememberPut` or `RememberCards`?

For clarity:
- RememberCards will remember the choice from PutCounter whether it gets the counter or not.
- RememberPut only remembers the choice if it actually gets the counter

In this case my gut says `RememberCards` but it may be affected by set notes or rulings - this wording seems new

Reply:
The problem here is the lack of targeting (this hasn't been done before)

Reply:
If you do keep ChooseCard you would need a cleanup

Reply:
There are a few other cards that do a thing to a card that got a counter without first targeting... and the Remember functions in CountersPutEffect don't require targeting... they look at the cards that the effect wants to (or did) put counters on whether they were chosen by the effect, defined, or targeted

Reply:
Thanks for all the comments, rememberCards did work. I guess Gix' Command can be simplified like this as well